src/Webscrape/WebscrapePage.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import Input from "../Input";
+import { submitNewURL } from "../api";
+import { useHistory } from "react-router-dom";
+import { withCookies } from "react-cookie";
+
+const WebscrapePage = ({ allCookies = {} }) => {
+	let history = useHistory();
+	const token = allCookies.sessionToken ? allCookies.sessionToken.token : null;
+	const [URL, setURL] = useState("");
+
+	const handleChange = event => {
+		setURL(event.target.value);
+	};
+
+	const submitURL = async () => {
+		try {
+			await submitNewURL(URL, token);
```

Comment:
may want to check whether the url exists before sending this request

Reply:
urmmm, like with an if statement?

Reply:
or using something similar to the above comment? getUserById(id, cookies.sessionToken.token);

Reply:
this has been added in!